src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -181,16 +179,19 @@ export class HostsComponent extends ListWithDetails implements OnInit {
     });
   }
 
-  getEditDisableDesc(selection: CdTableSelection): boolean | string {
-    if (selection?.hasSingleSelection) {
-      if (!selection?.first().sources.orchestrator) {
-        return $localize`Host editing is disabled because the selected host is not managed by Orchestrator.`;
+  getDisable(action: 'create' | 'edit' | 'delete', selection: CdTableSelection): boolean | string {
+    if (action === 'delete' || action === 'edit') {
+      if (!selection?.hasSelection) {
+        return true;
+      }
+      if (!selection.first().sources.orchestrator) {
```

Comment:
in the case of delete shouldn't you check if the orch is available in all selections?

Reply:
Fixed!